tools/querytee/proxy.go

```diff
@@ -40,7 +39,8 @@ type ProxyConfig struct {
 }
 
 func (cfg *ProxyConfig) RegisterFlags(f *flag.FlagSet) {
-	f.IntVar(&cfg.ServerServicePort, "server.service-port", 80, "The port where the query-tee service listens to.")
+	f.IntVar(&cfg.ServerServicePort, "server.service-port", 80, "The HTTP port where the query-tee service listens to.")
+	f.IntVar(&cfg.ServerGRPCServicePort, "server.grpc-service-port", 90, "The GRPC port where the query-tee service listens to HTTP over gRPC messages.")
```

Comment:
I would use the same default as Mimir.

```suggestion
	f.IntVar(&cfg.ServerGRPCServicePort, "server.grpc-service-port", 9095, "The GRPC port where the query-tee service listens to HTTP over gRPC messages.")
```

Reply:
@pracucci The Mimir HTTP default port is `8080`: https://github.com/grafana/mimir/blob/main/pkg/mimir/mimir.go#L319
Should the HTTP default also be updated, or better not as that is a breaking change?

Reply:
As you mentioned it would be a sort of breaking change, so I would keep it separated from this PR.